src/Microsoft.AspNetCore.Razor/Text/Internal/LineTrackingStringBuffer.cs

```diff
@@ -7,9 +7,9 @@
 using System.Text;
 using Microsoft.AspNetCore.Razor.Parser;
 
-namespace Microsoft.AspNetCore.Razor.Text
+namespace Microsoft.AspNetCore.Razor.Text.Internal
```

Comment:
Curious: Other than changes to match classes you're moving, why change any test `namespace`s?